frontend/components/panels/VaultLiquidationLimitsCheckPanel.vue

```diff
@@ -75,6 +75,7 @@
                 <span>DAI</span>
             </div>
         </div>
+        <!--
```

Comment:
Technically, you can trigger refetching of the vault, which will also get fresh limits